src/api/workListApi.ts

```diff
@@ -71,3 +71,41 @@ export const getWorkListItemsExpired = (): Promise<AxiosResponse<WorklistRequest
     method: 'GET',
   });
 };
+
+export const getWorkListItemsOwnership = (): Promise<AxiosResponse<WorklistRequest[]>> => {
+  return authenticatedApiRequest({
+    url: encodeURI(`${API_PATHS.guiBackendWorklistsPath}/worklists/types/dlr_resource_owner_request`),
+    method: 'GET',
+  });
+};
+
+export const refuseOwnershipRequest = (resourceIdentifier: string, comment: string) => {
```

Comment:
Er det støtte for tilbakemeldinger pr epost ennå ? hvis ikke burde vi kanskje disable funksjonen enn så lenge ?

Reply:
Da bør vi skru av funksjonaliteten også for DOI og rapporteringer. Står jo ikke at det er en tilbakemelding til den som rapporterte inn, bare at man kan skrive inn en kommentar om hvorfor det ble avvist. Ser for meg at det er nyttig uansett med tanke på dokumentasjon.

Reply:
"saved by the description" :-P  Ok,  vi lar vel den stå sånn da